applications/tari_dan_wallet_daemon/src/webrtc.rs

```diff
@@ -0,0 +1,171 @@
+//   Copyright 2023 The Tari Project
+//   SPDX-License-Identifier: BSD-3-Clause
+
+use std::sync::{Arc, Mutex};
+
+use anyhow::Result;
+use axum_jrpc::{JsonRpcAnswer, JsonRpcResponse};
+use reqwest::header::CONTENT_TYPE;
+use serde::Deserialize;
+use tari_shutdown::ShutdownSignal;
+use webrtc::{
+    api::APIBuilder,
+    data_channel::{data_channel_message::DataChannelMessage, RTCDataChannel},
+    ice_transport::{
+        ice_candidate::{RTCIceCandidate, RTCIceCandidateInit},
+        ice_server::RTCIceServer,
+    },
+    peer_connection::{configuration::RTCConfiguration, sdp::session_description::RTCSessionDescription},
+};
+
+#[derive(Deserialize, Debug)]
+struct Offer {
+    offer: String,
+}
+
+pub async fn handle_data(msg: String) -> Result<String> {
+    println!("webrtc received {msg}");
+    let (method, params) = match msg.split_once(' ') {
+        Some(a) => a,
+        None => (msg.as_str(), ""),
+    };
+    let url = "http://127.0.0.1:9000";
```

Comment:
should this be hardcoded ?

Reply:
No ;-) should take it from the config. Will address this in this or next PR.

Reply:
It's done :-)